data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1718,6 +1718,19 @@ spec:
                       the subnets. Leave the hosted zone unset to have the installer
                       create the hosted zone on your behalf.
                     type: string
+                  lbType:
+                    description: 'LBType allows user to set a load balancer type.
+                      When this field is set the default ingresscontroller will get
+                      created using the specified LBType. If this field is not set
+                      then the default ingress controller of LBType Classic will be
+                      created. Valid values are: * "Classic": A Classic Load Balancer
+                      that makes routing decisions at either   the transport layer
+                      (TCP/SSL) or the application layer (HTTP/HTTPS). See   the following
+                      for additional details:   https://docs.aws.amazon.com/AmazonECS/latest/developerguide/load-balancer-types.html#clb
+                      * "NLB": A Network Load Balancer that makes routing decisions
+                      at the   transport layer (TCP/SSL). See the following for additional
+                      details:   https://docs.aws.amazon.com/AmazonECS/latest/developerguide/load-balancer-types.html#nlb'
+                    type: string
```

Comment:
Isn't this a generated file?  I don't see how we can fix the double spaces.  I can add some text in `pkg/types/aws/platform.go` to say that this field is optional, in which case the default is "Classic".

Reply:
I updated the godoc and ran `go1.19 generate ./...` to regenerate `data/data/install.openshift.io_installconfigs.yaml`.

Reply:
I just did another push in which I removed indentation for the list items in the godoc in order to prevent the generated CRD yaml from having those double-spaces.

Reply:
I have some other suggestions other than just the indentation. Is that something that you feel is valid enough to be addressed?